interface/patient_file/merge_patients.php

```diff
@@ -313,27 +483,49 @@ function mergeRows($tblname, $colname, $source_pid, $target_pid)
    <input type='hidden' name='form_source_pid' value='0' />
   </td>
   <td>
-    <?php echo xlt('This is the chart that is to be merged into the main chart and then deleted.'); ?>
+    <?php echo xlt(
+        'This is the chart that is to be merged into the main chart 
+        and then deleted.'
+    ); ?>
   </td>
  </tr>
 </table>
-<p><input type='submit' class="btn btn-primary" name='form_submit' value='<?php echo xla('Merge'); ?>' /></p>
+<p><input type='submit' class="btn btn-primary" name='form_submit' 
+    value='<?php echo xla('Merge'); ?>' /></p>
 </div>
 </form>
 <div class="jumbotron">
-    <p class="font-weight-bold"><?php echo xlt('This utility is experimental. Back up your database and documents before using it!'); ?></p>
+    <p class="font-weight-bold"><?php echo xlt(
+        'This utility is experimental. 
+    Back up your database and documents before using it!'
+    ); ?></p>
 
 <?php if (!$PRODUCTION) { ?>
 <p><?php echo xlt('This will be a "dry run" with no physical data updates.'); ?></p>
 <?php } ?>
 
-<p><?php echo xlt('This will merge two patient charts into one.  It is useful when a patient has been duplicated by mistake.  If that happens often, fix your office procedures - do not run this routinely!'); ?></p>
-
-<p><?php echo xlt('The first ("target") chart is the one that is considered the most complete and accurate. Demographics, history and insurance sections for this one will be retained.'); ?></p>
-
-<p><?php echo xlt('The second ("source") chart will have its demographics, history and insurance sections discarded.  Its other data will be merged into the target chart.'); ?></p>
-
-<p><?php echo xlt('The merge will not run unless SSN and DOB for the two charts are identical. DOBs cannot be empty.'); ?></p>
+<p><?php echo xlt(
+    'This will merge two patient charts into one.  
+    It is useful when a patient has been duplicated by mistake.  
+    If that happens often, fix your office procedures - do not run this routinely!'
+); ?></p>
+
+<p><?php echo xlt(
+    'The first ("target") chart is the one that is considered 
+    the most complete and accurate. 
+    Demographics, history and insurance sections for this one will be retained.'
+); ?></p>
+
+<p><?php echo xlt(
+    'The second ("source") chart will have its demographics, 
+    history and insurance sections discarded.  
+    Its other data will be merged into the target chart.'
+); ?></p>
+
+<p><?php echo xlt(
+    'The merge will not run unless SSN and DOB for the two charts are identical.
+     DOBs cannot be empty.'
```

Comment:
For this line and 3 more above
ensure don't break up translation constant into multiple lines. need to make these one line no matter how long they are (or else will break in the translation pipeline)